### ISSUE TITLE
Basic GMM classifier with single mixtures

### ISSUE THREAD

**Issue by sravanareddy** [wellesleynlp/emilythesis#2]
1. Try out the scikit-learn example, http://scikit-learn.org/stable/auto_examples/mixture/plot_gmm_classifier.html#example-mixture-plot-gmm-classifier-py on the Iris data (which is apparently a 4 dimensional dataset on different classes of Iris flowers! -- http://scikit-learn.org/stable/auto_examples/datasets/plot_iris_dataset.html).
2. Then download the newly created `npytxt` directory in the CSLU corpus on tempest. The paper is unclear about the details, so I assume that the acoustic vector for each file is obtained by averaging all the vectors in that file. 

Write a program to classify among two or three accents, following the Iris example. Note that you have to make a bunch of changes to the scikit-learn code here in terms of reading and organizing the data from the files, and computing the average vectors (which is not an issue with the Iris one).

Try to get this done by Tuesday so that you can clarify any difficulties before we meet.

**Comment by sravanareddy** [comment]
Oh, also commit your code for these exercises into this repository as you work. Don't commit the data files.

**Comment by emilyahn** [comment]
This sounds awesome, I'll get started this weekend!

**Comment by sravanareddy** [comment]
The current code only considers the first 52 time slices (corresponding 520 ms) of each speaker. No wonder the accuracies were so poor. :)

**Comment by sravanareddy** [comment]
Ok, it seems one major issue is that the sklearn GMM code is incomplete. Notice that the .fit() method in https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/mixture/gmm.py does not make use of the category labels (y) at all! So we were basically just finding clusters in the data, not actually fitting the data to the accent labels.

That's ok, though, because we'll use a different methodology using multiple mixture components for each accent anyway. More details in issue #3, coming up.
